src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+@font-face {
+  font-family:  Avenir, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  color: #3D4E61;
+}
+
+.form {
+  position: relative;
+margin: 20px 8px 10px;
+padding: 0;
+}
+
+.form__input {
+  display: block;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)